index.html

```diff
@@ -38,6 +38,107 @@ <h2 class="connect">LET'S CONNECT</h2>
             </ul>
         </section>
 
+        <section class="portfolio">
+            <div class="project-cards">
+                <div class="cards card1">
+                    <img src="file:///C:/All%20stuff/Microverse/My-Portfolio/images/Snapshoot%20Portfolio.svg" alt="Snapshot Portfolio">
```

Comment:
All image you're using is not displaying to the page properly. It's because the path for the `src` is not correct. Please double-check and give the correct path for the image tag to display on the page. Here is what is displaying right now:

<img width="302" alt="image" src="https://user-images.githubusercontent.com/34578631/159096723-4f200259-a4cd-47a8-b5b0-e6bbd41b09ff.png">